phpunit/class-block-supported-styles-test.php

```diff
@@ -112,8 +112,14 @@ private function assert_content_and_styles_and_classes_match( $block, $expected_
 		$style_list = $this->get_attribute_from_block( 'style', $styled_block );
 
 		$this->assertEquals( self::BLOCK_CONTENT, $content );
-		$this->assertEquals( $expected_classes, $class_list );
-		$this->assertEquals( $expected_styles, $style_list );
+		$this->assertEqualSets(
+			explode( ' ', $expected_classes ),
+			explode( ' ', $class_list )
+		);
+		$this->assertEquals(
```

Comment:
I presume this isn't `assertEqualSets` as `foo:bar; foo:bart` is not the same as `foo:bart; foo:bar`. Is that correct?

Reply:
It was actually unintentional 🤦 sorry about that. 

But now that you say that, style property order could matter in some circumstances for things like browser fallbacks. So perhaps it should stay that way?

Reply:
@TimothyBJacobs Not just browser fallbacks. Although I'd consider it questionable practice, you can do something like `padding: 4px; padding-left: 8px`, where the left padding will be different depending on the property order.